udata_front/theme/gouvfr/templates/macros/metadata.html

```diff
@@ -20,7 +20,8 @@
 <meta name="static-root" content="{{ static('', _burst=False) }}" />
 <meta name="api-root" content="{{ url_for('api.root') }}" />
 <meta name="api-2-root" content="{{ url_for('apiv2.root') }}" />
-<meta name="admin-root" content="{{ url_for('admin.index') }}" />
+<meta name="admin-root" content="{{ beta_admin_url_for(path='/', fallback_path='/') }}" />
```

Comment:
I think fallback_path shouldn't have a `/` to prevent duplicate `//`
```suggestion
<meta name="admin-root" content="{{ beta_admin_url_for(path='/', fallback_path='') }}" />
```